modules/va_forms/app/controllers/va_forms/application_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module VaForms
+  class ApplicationController < ::OpenidApplicationController
```

Comment:
We don't need to inherit from OpenIDApplicationController unless we were doing an Oauth API end point, I would instead look at what VBA Documents inherits from and based it from there.